src/Tribe/Views/V2/View.php

```diff
@@ -1585,6 +1585,20 @@ protected function get_breadcrumbs() {
 			}
 		}
 
+		// Setup breadcrumbs for when it's featured.
+		if ( $is_featured = tribe_is_truthy( $this->context->get( 'featured', false ) ) ) {
+			$non_featured_link = tribe_events_get_url( [ 'featured' => 0 ] );
+
+			$breadcrumbs[] = [
+				'link'  => $non_featured_link,
```

Comment:
Why aren't we using `tribe_get_events_link()` here?

Reply:
That is the old method. We need to figure out how to deprecated a lot of those or route them correctly.